package.json

```diff
@@ -161,15 +161,14 @@
     "jasmine-core": "^2.99.1",
     "jasmine-marbles": "^0.4.1",
     "jasmine-spec-reporter": "~4.2.1",
-    "json-server": "^0.14.2",
+    "json-server": "^0.16.3",
```

Comment:
@leepavlouhmcts - just querying whether this needs to go up to `0.16.3` or can be left at `0.14.2`?

Reply:
if i put it back it runs but i get 2 additiional vulnerabilities, 1 high and one moderate which i have to add to resolutions

Reply:
Resolving as OK, following confirmation from @leepavlouhmcts.